src/Microsoft.DocAsCode.App/Microsoft.DocAsCode.App.csproj

```diff
@@ -13,7 +13,7 @@
   </Target>
 
   <ItemGroup>
-    <Content Include="templates/**" CopyToOutputDirectory="PreserveNewest" PackageCopyToOutput="true" />
+    <Content Include="templates/**" CopyToOutputDirectory="PreserveNewest" PackageCopyToOutput="true" PackagePath="contentFiles/any/any/templates"/>
```

Comment:
Does this change breaks the `docfx` package shipped as a global tool?

Reply:
This PR change don't affects `docfx.nupkg` contents.   
(Because `docfx` reference `Microsoft.DocAsCode.App` as `ProjectReference`)
So there is no breaking changes. 

---

I though there are 3 execution type to run docfx toolset.

1. Run `docfx`  as .NET global tool (and .NET local tool)
2. Run `docfx` Self-Contained exe that is published on [GitHub Release Page](https://github.com/dotnet/docfx/releases)
3. Run custom application that use `Microsoft.DocAsCode.App` package.

`1.` and `2.`  behaviors are guaranteed by there is no output content changes.
And `3.` is tested on my local environment. and confirmed no behavior changes.